spec/controllers/graders_controller_spec.rb

```diff
@@ -503,6 +507,40 @@
           expect(@grouping1.tas.size).to eq 0
           expect(@grouping2.tas.size).to eq 0
         end
+        it 'and only inactive graders (and multiple groupings) are selected' do
+          post_as @instructor,
+                  :global_actions,
+                  params: {
+                    course_id: course.id, assignment_id: @assignment.id, global_actions: 'random_assign',
+                    groupings: [@grouping1, @grouping2, @grouping3], graders: [@ta4, @ta5],
```

Comment:
See my above comment about the `graders` parameter (and check this throughout all test cases)